SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs

```diff
@@ -107,7 +122,11 @@ public partial class SettingsPageViewModel : ViewModelBase
 
     protected override void OnPropertyChanged(PropertyChangedEventArgs e) {
         base.OnPropertyChanged(e);
-
+        if (e.PropertyName == nameof(CurrentLocalization))
+        {
+            _localizationService.Translate(CurrentLocalization);
+            Config.CurrentLanguageSelected = CurrentLocalization.Name;
+        }
```

Comment:
I know I said two things but I can't think how to handle the second so we can leave it for now and I'll create an issue to come back to it sometime in the future.  So one last thing then this is good to go. 

Could we take the highlighted lines out and instead use th the specific property change value for CurrentLanguageSelected?

If you wanna just copy and paste the function it'd be this:
```
    partial void OnCurrentLocalizationChanged(CultureInfo value)
    {
        _localizationService.Translate(value);
        Config.CurrentLanguageSelected = value.Name;
    }
```